lib/unimatrix/authorization/parser.rb

```diff
@@ -2,13 +2,14 @@ module Unimatrix::Authorization
 
   class Parser
 
-    def initialize( content = {} )
+    def initialize( content = {}, request_path = {} )
```

Comment:
Do you know why these arguments are given default values?  Seems that this filter doesn't do anything unless both of these are present, should they be required arguments?